repository/test.lua

```diff
@@ -183,7 +183,8 @@ forEachOrderedElement(pkgs, function(package)
 	local command = "terrame -package "..package.." -projects"
 	runCommand(command)
 
-	local proFinalTime, difference = profiler:stop("REPOSITORY_PROJECT_")
+	local proFinalTime = profiler:stop("REPOSITORY_PROJECT_").strTime
+	local difference = profiler:stop("REPOSITORY_PROJECT_").time
```

Comment:
Do not call stop twice with the same argument without starting again. Better to show a warning in this case.